navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationContract.java

```diff
@@ -0,0 +1,48 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import com.mapbox.services.api.directions.v5.models.DirectionsRoute;
+import com.mapbox.services.commons.models.Position;
+
+public interface NavigationContract {
+
+  interface View {
```

Comment:
at some point, no requirement for this PR, we could look into decoupling this interface into smaller interfaces eg. SummaryView which encapsulates the methods below or a RecenterView that encapsulates hide/show.

Reply:
@tobrun Yeah definitely, would you recommend each custom view having their own presenter as well?